scout/server/blueprints/variants/templates/variants/components.html

```diff
@@ -96,15 +96,24 @@
           </span>
         {% endif %}
       </a>
-      {% if variant.panels|count > 0 %}
+
+      {% set ns = namespace(panel_count = 0) %}
+      {% for panel in variant.case_panels %}
+        {% if not panel.removed %}
+          {% set ns.panel_count = ns.panel_count + 1 %}
```

Comment:
Just a question, can't |rejectattr('removed') used to count as well?

Reply:
It doesn't work with the jinja `length` operation after `rejectattr`, unfortunately, but I think it should work instead of the `if not panel.removed` certainly!

Reply:
Lets see the reason it doesn't work with `length` appears to be that `rejectattr` returns a generator object. So perhaps `|rejectattr|list|length` could work!?